### PR TITLE
Newcomers_Guide.rst: Link style guidelines

### PR DESCRIPTION
**Pull request by IpshitaC**
Added instructions about code documentation and styling
Linked Writing_Documentation.rst and PEP8 guidelines for references

Fixes https://github.com/coala/coala/issues/4255

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -210,6 +210,20 @@ Now you can start working on it.
     Before starting to write your first commit, check out this
     link: `Writing good commits <http://coala.io/commit>`_.
 
+.. seealso::
+
+    An important part of working on issues is documenting your work
+    in such a way that it is easy for others to read and understand.
+    A lot of Newcomer issues involve improving documentation.
```
Comment (Monal5031) [inline docs/Developers/Newcomers_Guide.rst]: Shouldn't it be `Newcomers` ?

Comment (yuxiangdai) [inline docs/Developers/Newcomers_Guide.rst]: `Newcomer` is fine, otherwise it would be `Newcomers' `

**Comment by shreyans800755** [comment]
@IpshitaC Changes looks good.

Shortlog looks good. Can you please update description to something like this:

> Adds instructions about code documentation and styling.
Linked Writing_Documentation.rst and PEP8 guidelines for references.

> Fixes coala#4255

**Comment by IpshitaC** [comment]
@shreyans800755 done! :+1:

**Comment by shreyans800755** [comment]
@IpshitaC Sorry just one more change I missed.
Just change `Linked` to `Links` in the commit.

**Comment by IpshitaC** [comment]
@shreyans800755 no issues, done! :+1:

**Comment by shreyans800755** [comment]
ack 045c4aa

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -210,6 +210,20 @@ Now you can start working on it.
...
+
+    * For more information about writing good documentation,
+      please check the following link: `Writing Documentation <https://api.coala.io/en/latest/Developers/Writing_Documentation.html>`_
+
+    * For more information about how to style Python code
+      according to the PEP8 code style convention,
```
Comment (satwikkansal) [inline docs/Developers/Newcomers_Guide.rst]: *conventions :)

Comment (satwikkansal) [inline docs/Developers/Newcomers_Guide.rst]: PEP8 coding conventions sounds better to me, anyways it's up to you :)

**Comment by IpshitaC** [comment]
@Udayan12167 Your views on comments ? How should I modify my PR?

File: docs/Developers/Newcomers_Guide.rst (hunk 2 above)
Comment (shreyans800755) [inline docs/Developers/Newcomers_Guide.rst]: I think `style` is more suitable over here as official docs of PEP8 also refers it as `style`. https://www.python.org/dev/peps/pep-0008

**Comment by Udayan12167** [comment]
ack https://github.com/coala/coala/pull/4318/commits/045c4aaeea6c2af5146edb500a1c640c1057355b

**Comment by Udayan12167** [comment]
@rultor merge

**Pull request merged**